src/test/java/wolox/training/TrainingApplicationTests.java

```diff
@@ -8,4 +10,8 @@
 @SpringBootTest
 public class TrainingApplicationTests {
 
+    @Test
+    public void dummy() {
+        Assertions.assertThat(true).isTrue();
```

Comment:
You can leave this test empty and it will also work :blush:

Reply:
It generates something like a test fail because the class has no annotated methods with `@Test`, so I put it in there.

Reply:
It fails if you put ` public void contextLoad() {
}` ?

Reply:
Yes, it fails because the absence of test methods